tests/foreman/ui/test_host.py

```diff
@@ -498,7 +498,7 @@ def test_positive_read_from_details_page(session, module_host_template):
         assert session.host.search(host_name)[0]['Name'] == host_name
         values = session.host.get_details(host_name)
         assert values['properties']['properties_table']['Status'] == 'OK'
-        assert values['properties']['properties_table']['Build'] == 'Pending installation'
+        assert values['properties']['properties_table']['Build'] == 'Pending installation clear'
```

Comment:
This is actually shown as status "Pending installation" and a link "clear" next to it. Wouldn't it be better to make the get_details method read the status correctly?

Reply:
Will look at it.

Reply:
this was discussed here: https://github.com/SatelliteQE/robottelo/pull/7845#discussion_r443466606

Reply:
If you think doing it properly is unnecessary effort, I'm OK with merging as it is.